openapi/workflow_execution_service.openapi.yaml

```diff
@@ -813,12 +844,18 @@ components:
         runs:
           type: array
           items:
-            $ref: '#/components/schemas/State'
+            anyOf:
+              - $ref: '#/components/schemas/RunStatus'
+              - $ref: '#/components/schemas/RunSummary'
```

Comment:
I like this approach, since it does not break the API, but strongly suggests to implementors to use the new approach

Reply:
Agree